source/connecting/atlas-to-atlas.txt

```diff
@@ -19,6 +19,16 @@ This page provides instructions to connect Atlas clusters using
 Atlas cluster, see :ref:`Connect to a Database Deployment
 <atlas-connect-to-deployment>` 
 
+Considerations 
+--------------
+
+- Both the source and destination Atlas clusters must use MongoDB 6.0 or 
+  later. 
+- ``mongosync`` supports replica sets and sharded clusters.
+- ``mongosync`` does not support Atlas shared clusters or serverless
```

Comment:
[Suggestion]: I like to bold the "not" in phrases like this to make sure the reader sees it.